tests/jvm/src/test/resources/metacp.expect

```diff
@@ -189,13 +94,13 @@ Schema => SemanticDB v3
 Uri => com/javacp/Javacp.class
 Text => empty
 Language => Java
-Symbols => 56 entries
+Symbols => 66 entries
 
 Symbols:
 _root_. => package _root_
 _root_.com. => package com
 _root_.com.javacp. => package javacp
-_root_.com.javacp.Javacp# => class Javacp[A <: CharSequence with Serializable, B <: Object].{+30 decls}
+_root_.com.javacp.Javacp# => class Javacp[A <: CharSequence with Serializable, B <: Object].{+31 decls}
```

Comment:
Why is it just +1 decl when you added both a static class and a static method?

Reply:
Good catch, I've updated the logic to include direct inner classes as ClassInfo.declarations.